src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -444,27 +445,21 @@ private protected override void SaveModel(ModelSaveContext ctx)
                 ctx.SaveNonEmptyString(colName);
         }
 
-        ~TensorFlowTransformer()
+        public void Dispose()
         {
-            Dispose(false);
-        }
+            if (_isDisposed)
+                return;
```

Comment:
💭 I would have liked to see `Session` have a more idiomatic dispose behavior so this state tracking wasn't necessary, but I wasn't able to prove it to be the case.